tsconfig.json

```diff
@@ -0,0 +1,40 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "allowJs": true,
+    "skipLibCheck": true,
```

Comment:
Also can we add ESLint for the typescript

https://github.com/typescript-eslint/typescript-eslint

Reply:
For the eslint let's go for the one that doesn't place `;` at the end of every line, and it can auto format for you.

Use this package for the eslint config: https://www.npmjs.com/package/eslint-config-standard-with-typescript

Reply:
Done! Add `lint` and `Prettier` @NanopassKai